src/test/resources/log4j2-test.properties

```diff
@@ -13,7 +13,7 @@ appender.file.layout.type=PatternLayout
 appender.file.layout.pattern=[%-5level] %d{yyyy-MM-dd HH:mm:ss.SSS} [%t] %c{1} - %msg%n
 
 
-rootLogger.level = warn
+rootLogger.level = info
```

Comment:
wdyt about making this more targeted like the entries in this file down below? Its possible to set the log level by classname if you are targeting a specific log line in a test case.

Reply:
done